content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,635 @@
+client is owned by a particular team or group, the same can be done for specific
+checks. For instance:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  }
+}
+{{< /highlight >}}
+
+### Client Configuration
+
+A client configuration can include a contact to use.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+In this section we will be showcasing how contact routing interacts depending on the different possible configurations.
+
+## Global Configuration
+
+For the examples we'll be working with we want to define what our global handlers will be.
+
+In the following section we'll be going over different scenarios and showcase how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and Pagerduty as our example handlers.
+
+Bellow is the default configuration for the handlers we'll be working with in our examples:
+
+{{< highlight json >}}
+{
+  "slack": {
+    "webhook_url": "https://hooks.slack.com/services/foo/bar/foobar",
+    "username": "sensu",
+    "channel": "#alerts",
+    "timeout": 10
+  },
+  "email": {
+    "smtp": {
+      "address": "smtp.example.com",
+      "port": 587,
+      "openssl_verify_mode": "none",
+      "enable_starttls_auto": true,
+      "authentication": "plain",
+      "user_name": "postmaster@example.com",
+      "password": "SECRET"
+    },
+    "to": "default@example.com",
+    "from": "noreply@example.com",
+    "timeout": 10
+  },
+  "pagerduty": {
+    "service_key": "someservicekey ",
+    "timeout": 10
+  }
+}
+{{< /highlight >}}
+
+## Single Handler
+
+### Single Handler with Single Matching Contact
+
+This configuration we have a single handler (email) a contact (support) configured. The support contact has a configuration override for email that will change the default "to" email address from "default@example.com" to "support@sensuapp.com".
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  },
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+Because the contact configuration matches the handler being used, any event generated for this check will use this contact's configuration.
+
+TODO: ADD IMAGE
+<img alt="Single Handler with Single Matching Contact" title="Single Handler with Single Matching Contact" src="/images/contact-routing-single-handler-single-matching-contact.png">
```

Comment:
```suggestion
![Single Handler with Single Matching Contact](/images/contact-routing-single-handler-single-matching-contact.png)
```
^ Would be the way to insert an image using markdown. A better way would be:

`![Single Handler with Single Matching Contact][4]`

^ As this would keep with our convention of adding links at the end of a file, rather than adding them inline.